src/applications/personalization/view-dependents/util/index.js

```diff
@@ -8,3 +11,25 @@ export async function getData(apiRoute, options) {
     return error;
   }
 }
+
+export function splitPersons(persons) {
+  const onAwardPeeps = [];
+  const notOnAwardPeeps = [];
```

Comment:
These could be named a little more semantically. Something like `dependentsOnAward` and `dependentsNotOnAward`.